tests/plotting/test_widgets.py

```diff
@@ -150,10 +150,12 @@ def test_widget_slider(uniform):
     p.add_slider_widget(callback=func, rng=[0, 10], style="modern", pass_widget=True)
     p.close()
 
-    p = pyvista.Plotter()
-    p.add_mesh_threshold(uniform, invert=True)
-    p.add_mesh(uniform.outline())
-    p.close()
+    if pyvista.vtk_version_info >= (9,):
+        # Invert not support for VTK8.1.2
+        p = pyvista.Plotter()
+        p.add_mesh_threshold(uniform, invert=True)
+        p.add_mesh(uniform.outline())
+        p.close()
```

Comment:
I guess we can improve the coverage of the test.
```suggestion
        p.close()
    else:
        # Invert not support for VTK8.1.2
        with pytest.raises(ValueError):
            p = pyvista.Plotter()
            p.add_mesh_threshold(uniform, invert=True)
```

Reply:
I'm actually going to revert this. This is hard to test for as the valueerror is being raised in a callback in a different event loop.

Reply:
OK. Let's skip this using https://github.com/pyvista/pyvista/pull/3750#discussion_r1058441168 .